### DISCUSSION TITLE
How do you increase the request size when NOT using bodyParser (everything above 1MB fails)

### DISCUSSION THREAD

**Discussion by tomitrescak** [vercel/next.js#15036] [Help]
I have an API route that uses graphql and ''apollo-server-micro'. This route DOES NOT use body parser, i.e.:

```
export const config = {
  api: {
    bodyParser: false
  }
};
```

BUT, when I try to send any request larger than 1MB it starts to complain that the "Body Parser is missing". Is this a bug?

**Comment by bentron2000** [comment]
Bump. I'd like to know the answer to this also. @tomitrescak did you find a solution?
